src/uTensor/core/operatorBase.cpp

```diff
@@ -15,26 +15,26 @@ DEFINE_ERROR(OperatorIOSizeMismatchError);
 //}
 // The preferred interface
 OperatorBase::OperatorBase(TensorMapInterface* inputs) : _p_inputs(inputs) {
-  Context* ctx = Context::get_default_context();
```

Comment:
Thanks for commenting these out, it's a placeholder for the operator to request the tensor be migrated into the current context. Super helpful for multi-chip